spy_terminal.py

```diff
@@ -16,6 +17,8 @@
 
 '''This main function calls all other functions and will maintain the 
 function as 'active' until the user exits it.'''
+
+
 def main():
```

Comment:
In Python, it's often best to start the name of a function with a verb because functions are action-oriented. Perhaps `run_program()` would be more appropriate here